pkg/util/pki/pki_test.go

```diff
@@ -1,61 +0,0 @@
-package pki
-
-// Copyright (c) Microsoft Corporation.
-// Licensed under the Apache License 2.0.
-
-import (
-	"crypto/tls"
-	"crypto/x509"
-	"fmt"
-	"net/http"
-	"os"
-	"testing"
-)
-
-func TestGetTlsConfig(t *testing.T) {
-	if os.Getenv("ARO_SKIP_PKI_TESTS") != "" {
```

Comment:
Should we remove this env var as part of this PR?

Reply:
Yes. Done. Thanks.